samples/androidNativeActivity/README.md

```diff
@@ -8,5 +8,6 @@ See https://developer.android.com/studio/index.html for more details on Android
 
 To build use `ANDROID_HOME=<your path to android sdk> ../gradlew buildApk`.
 
-Run `$ANDROID_HOME/sdk/platform-tools/adb install -r build/outputs/apk/androidNativeActivity-arm-debug.apk`
+Run `$ANDROID_HOME/platform-tools/adb install -r build/outputs/apk/debug/androidNativeActivity-arm-debug.apk`
```

Comment:
> `androidNativeActivity-arm-debug.apk`

On my machine it is `androidNativeActivity-debug.apk`

Reply:
OK